ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
```

Comment:
Should we somehow discourage people from asking usage questions that are more suited to a place like stack overflow? I've come across a number of those in the past.